packages/atomic/src/components/atomic-result-list/atomic-result-list.pcss

```diff
@@ -23,115 +23,113 @@
 }
 
 /* CAREFUL! These styles aren't protected by a shadow DOM. */
-atomic-result-list {
```

Comment:
This is the only non-formatting change in this file. I removed `atomic-result-list` since it's no longer needed.

Reply:
We can remove the comment?

Reply:
Woops, I completely missed that 🤦‍♂️